lib/bike_brigade/riders.ex

```diff
@@ -139,18 +139,15 @@ defmodule BikeBrigade.Riders do
     query =
       from c in Campaign,
         as: :campaign,
-        join: cr in CampaignRider,
```

Comment:
Nice! Thanks for catching this and bringing back the old code.

I think that we could probably refactor this entire function out and use `get_itinerary` in the place where this is used BUT this is out of the scope of this PR.